buildconfig/CMake/GNUSetup.cmake

```diff
@@ -115,3 +115,13 @@ if( CMAKE_COMPILER_IS_GNUCXX )
     set( CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "${CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS} -static-libgcc -static-libstdc++" )
   endif()
 endif()
+
+option (COLORED_OUTPUT "Always produce ANSI-colored output (GNU/Clang only)." TRUE)
+
+if (${COLORED_OUTPUT})
+    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
```

Comment:
The same can be done for `STREQUAL`: `if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")` is equivalent but is easier to read with the quotes and brackets.

Reply:
Can I use the `CMAKE_COMPILER_IS_GNUCXX` variable instead of doing the STREQUAL for GNU?

Reply:
https://cmake.org/cmake/help/v3.14/variable/CMAKE_COMPILER_IS_GNUCXX.html?highlight=cmake_compiler_is_gnucxx seems to recommend using `CMAKE_CXX_COMPILER_ID`